addon/adapters/pouch.js

```diff
@@ -365,9 +365,9 @@ export default DS.RESTAdapter.extend({
 
     //Delete the children
     let deletedChildren = [];
-    snapshot.eachRelationship(function(name, descriptor) {
+    snapshot.eachRelationship((name, descriptor) => {
       if(descriptor.kind === 'hasMany' && descriptor.options['cascadeDestroy']) {
-        let mappedPromise = snapshot.record.get(name).then(function(childRecords) {
+        let mappedPromise = snapshot.record.get(name).then(childRecords => {
```

Comment:
Personal preference, but I might actually go with the parentheses for params here, just to be consistent with [line 368](https://github.com/nolanlawson/ember-pouch/pull/145/files/cd46f568efccb5c9d0214281162535edecc13189..eee182892d8b2c2977e5c392381bb86b0d3e2a16#diff-8bef95280660598dc2f4437f447292e2R368):

```
let mappedPromise = snapshot.record.get(name).then((childRecords) => {
```

Preference, not necessary.